configsuite/config.py

```diff
@@ -33,19 +33,26 @@ def __init__(
         schema,
         layers=(),
         extract_validation_context=lambda snapshot: None,
+        extract_transformation_context=lambda snapshot: None,
     ):
         assert_valid_schema(schema)
         self._layers = tuple(
             [copy.deepcopy(layer) for layer in tuple(layers) + (raw_config,)]
         )
-        self._cached_raw_config = None
         self._schema = copy.deepcopy(schema)
         self._extract_validation_context = extract_validation_context
+        self._extract_transformation_context = extract_transformation_context
+
         self._snapshot = None
-        self._valid = None
-        self._errors = None
-        self._readable = None
-        self._validate()
+        self._cached_merged_config = None
```

Comment:
is it necessary to declare _cached_merge_config = None at this point since its explicitly declared further down?

Reply:
Well spotted. This should have been removed in the re-factoring...